mxcube3-server

```diff
@@ -5,4 +5,4 @@ monkey.patch_all()
 from mxcube3 import app, socketio
 
 if __name__ == '__main__':
-    socketio.run(app, host='localhost', port=8081)
+    socketio.run(app, host='192.168.56.101', port=8081)
```

Comment:
Yes, I agree with yesterday's PR.
Ignore my changes. It was like that due to my vm configuration.